tests/auth_tests/models/__init__.py

```diff
@@ -15,5 +16,6 @@
     'CustomUser', 'CustomUserWithoutIsActiveField', 'CustomPermissionsUser',
     'CustomUserWithFK', 'Email', 'ExtensionUser', 'IsActiveTestUser1',
     'MinimalUser', 'NoPasswordUser', 'UUIDUser', 'CustomUserNonUniqueUsername',
-    'IntegerUsernameUser', 'UserWithDisabledLastLoginField',
+    'IntegerUsernameUser', 'UserWithDisabledLastLoginField', 'Proxy',
+    'UserProxy',
```

Comment:
Should I re-order the `__all__`? I'm not sure what is the logical order used currently...

Reply:
I did... let met know if I shouldn't have :)